### PR TITLE
Fixed bug in get_result_from_folder in spyking circus

### PR DESCRIPTION
**Pull request by colehurwitz**

### CODE REVIEW

**Comment by colehurwitz** [comment]
Can i merge this @alejoe91 @samuelgarcia 

Spyking had a bug with get_result_from_folder

File: README.md [hunk 1]
```diff
@@ -67,6 +65,32 @@ sorting_KS = st.sorters.run_kilosort(recording')
...
+    .
+    SorterClass = st.sorters.sorter_dict[sorter_name_or_class]
+    sorter = SorterClass(recording=recording, output_folder=output_folder, grouping_property=grouping_property,
+                     parallel=parallel, debug=debug, delete_output_folder=delete_output_folder)
+    sorter.set_params(**params)
+    sorter.run()
+    sortingextractor = sorter.get_result()
+    return sortingextractor
+```
+
```
Comment (samuelgarcia) [inline README.md]: I think this is too much detaiol for a REAME main page.

**Comment by samuelgarcia** [comment]
This PR mix:
  * GUi params
  * SC bugfix
  * doc in readme

You should split. No ?

**Comment by colehurwitz** [comment]
I can remove the README changes if you think it is too much. I was talking with Alessio about how multiple users have asked me how to get the params used with a sorter and I always show them the object oriented approach to sorting.

**Comment by colehurwitz** [comment]
> You should split. No ?

Yes, I accidently combined my GUI pull request with this one... I can split

File: spiketoolkit/sorters/spyking_circus/spyking_circus.py [hunk 2]
```diff
@@ -107,5 +107,5 @@ def _run(self,  recording, output_folder):
 
     @staticmethod
     def get_result_from_folder(output_folder):
-        sorting = se.SpykingCircusSortingExtractor(output_folder / 'recording')
+        sorting = se.SpykingCircusSortingExtractor(output_folder)
```
Comment (samuelgarcia) [inline spiketoolkit/sorters/spyking_circus/spyking_circus.py]: This pass test locally ?

**Comment by colehurwitz** [comment]
This fails in a lot of ways! I will close this PR

**Pull request closed without merging**